pkg/imgpkg/cmd/imgpkg.go

```diff
@@ -69,6 +70,24 @@ func NewImgpkgCmd(o *ImgpkgOptions) *cobra.Command {
 	// This configurations forces all nodes to do not accept extra args, but the completion requires 1 extra arg
 	cmd.AddCommand(NewCompletionCmd())
 
+	err := cmd.LocalFlags().MarkHidden("azure-container-registry-config")
+	if err != nil {
+		//TODO: wrap around a useful err msg
+		panic(err.Error())
+	}
+
+	cmd.PersistentPreRunE = func(subCmd *cobra.Command, args []string) error {
```

Comment:
i would throw in a quick panic if cmd.PersistentPreRunE is already defined. to make sure we dont override things accidently.

Reply:
good idea. done